test/bases.py

```diff
@@ -104,7 +123,7 @@ def _get_items_partial(self, item_pool, missing_item):
 
 
 class WorldTestBase(unittest.TestCase):
-    options: typing.Dict[str, typing.Any] = {}
+    options: typing.Dict[str, typing.Any] = DefaultOptionsDict()
```

Comment:
maybe this to more clearly illustrate the design intent?
```suggestion
    options: typing.ClassVar[typing.Dict[str, typing.Any]] = DefaultOptionsDict()
```

Reply:
This is intentional, because assigning options dynamically on instance level (in setUp) is an awkward but legit.

Reply:
Considering Zillion is the only game I've found currently merged that reassign options while a test is running (worlds/zillion/test/TestOptions.py), this would not break too much.

Reply:
> This is intentional, because assigning options dynamically on instance level (in setUp) is awkward but legit.
> 
> (Your suggestion is technically a breaking API change.)

Unfortunate. It means that to pass modern linters like ruff, you have to use MappingProxyType which is ugly af

Reply:
MappingProxyType makes the intention clear, so i am not sure it really is *ugly*.

I have not looked at type checking since originally opening the PR, feel free to point out if anything needs updating.

Either way, I stand by my opinion that making it a ClassVar is needlessly restricting the WorldTestBase.